packages/project-build/src/schema/props.ts

```diff
@@ -30,6 +30,11 @@ export const Prop = z.union([
     type: z.literal("asset"),
     value: z.string(),
   }),
+  z.object({
+    ...baseProp,
+    type: z.literal("string[]"),
+    value: z.array(z.string()),
+  }),
```

Comment:
@istarkov @TrySound 

Asking just in case: is it ok to add a new prop type "string[]"? Are there any concerns?